documentation/using-hyperview/sensor-management/index.rst

```diff
@@ -0,0 +1,76 @@
+*****************
+Sensor Management
+*****************
+
+============
+Key concepts
+============
+
+Sensor categories
+-----------------
+Sensor data is one of the primary data and analytics pipelines in Hyperview. Sensors have to be associated with assets and have the following categories:
+
+1. Direct: these sensors are directly attached to assets. For example, the Total Output Power sensor is read from a rack PDU. Data collectors update direct sensors.
+2. Linked: these sensors are sourced from other assets. For example, a temperature sensor discovered on an environmental monitor may provide information on the temperature of a rack.
+3. Computed: These sensors are computed from data available in the platform, For example, available rack space in a rack or from other read data or sensors; for instance, Output KVA and power factor can allow you to calculate total output power.
```

Comment:
"...the platform. For example,..." instead of "...the platform, For example,..."

Reply:
Thanks! Updated